client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -12,28 +13,209 @@ import { connect } from 'react-redux';
  */
 import {
 	fetchEmailSettings,
+	emailSettingChange,
+	emailSettingsSubmitSettings,
 } from 'woocommerce/state/sites/settings/email/actions';
+import {
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+	emailSettingsSaveRequest,
+	isSavingEmailSettings,
+	emailSettingsSubmitSettingsError,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import { errorNotice, successNotice } from 'state/notices/actions';
+import CustomerNotification from './components/customer-notification';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+import List from 'woocommerce/components/list/list';
+import ListHeader from 'woocommerce/components/list/list-header';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+
+const originNotifications = [
+	{
+		field: 'email',
+		option: 'woocommerce_email_from_name',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inboxes \'from\' this name.' ),
+	},
+	{
+		field: 'email',
+		option: 'woocommerce_email_from_address',
+		title: translate( 'From address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **87** times:
`translate( 'Address' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).